using new husky plugin to control wheels separately

mahmoudakl opened the pull request.

mahmoudakl committed:
Adjusted experiment timestep

mahmoudakl committed:
modified Brain2Robot and saved wheel_speeds to csv

mahmoudakl committed:
modified Brain2Robot and saved wheel_speeds to csv

mahmoudakl committed:
modified Brain2Motor TF to control wheel separately

mahmoudakl committed:
Merge branch 'master' of https://github.com/mahmoudakl/Floreano

S-Hedgehog commented:
Ah, danke.

Exakt das Selbe das habe ich gerade auch gemacht und wollte es testen bevor ich es commite. [😃]


________________________________
From: Mahmoud Akl <notifications@github.com>
Sent: Tuesday, September 5, 2017 13:53
To: S-Hedgehog/Floreano
Cc: Subscribed
Subject: [S-Hedgehog/Floreano] using new husky plugin to control wheels separately (#2)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/S-Hedgehog/Floreano/pull/2

Commit Summary

  *   Adjusted experiment timestep
  *   modified Brain2Robot and saved wheel_speeds to csv
  *   modified Brain2Robot and saved wheel_speeds to csv
  *   modified Brain2Motor TF to control wheel separately
  *   Merge branch 'master' of https://github.com/mahmoudakl/Floreano

File Changes

  *   M Experiments/floreano/Brain2Motor.py<https://github.com/S-Hedgehog/Floreano/pull/2/files#diff-0> (7)

Patch Links:

  *   https://github.com/S-Hedgehog/Floreano/pull/2.patch
  *   https://github.com/S-Hedgehog/Floreano/pull/2.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/S-Hedgehog/Floreano/pull/2>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AMkZQPhv6sfme4st0Aj8d03XuR2SYSt2ks5sfTYggaJpZM4PM1Xf>.

Commit:
- modified Brain2Robot
- changed handling of visual input to refelect more states
- made visual processing more efficient

Commit:
- changed wheel_speed_error calculation
- changed Brain2Motor:
-- neuron fireing rates are mapped to the interval [0.0, 1.0]
-- all negative wheel speeds are set to 0, since any instance that would produce negative speeds would have a fitness of 0 during evolution
--adjusted multiplicators to produce smoothen operation
- added global variable last_position to calculate the total distance traveled by the robot for testing and calculation of fitness for evolution

Commit:
- added experimental version of the evolutionary learning functionality for jupyter notebook

Commit:
added ipynb

Commit:
changed brain parameters

Commit:
first attempt at changing the brain architecture

Commit:
changed default tfs to handle different brain architecture

Commit:
fixes

corrected mistakes in code

Commit:
first attempt at mapping groups for sensor inputs
loads up without errors
neuron parameters need change to spike

Commit:
changed parameters and started changing the learning script

Commit:
getting error on reset, more testing needed

Commit:
bug fix

Commit:
cleanup

Commit:
Merge branch 'master' into master

Commit:
Merge branch 'master' into master
